thorgame/dsr209/thor_codingame.cpp

```diff
@@ -2,15 +2,13 @@
 //
```

Comment:
Start with the title of the game.

Reply:
I have updated this to include the title of the game at the beginning.